components/motor/gpio/motor_encoder.go

```diff
@@ -531,6 +598,9 @@ func (m *EncodedMotor) Stop(ctx context.Context, extra map[string]interface{}) e
 	m.stateMu.Lock()
 	m.state.goalRPM = 0
 	m.state.regulated = false
+	// if m.loop != nil {
+	// 	m.loop.Stop()
+	// }
```

Comment:
Ideally I would like to have this or something so when you stop the motor the control loop doesn't just restart it since it's trying to maintain the current position, but as we saw @randhid it caused issues with the autotuning process so wondering if anyone has suggestions

Reply:
we could set the constant value to ignore the control loop, a speed of zero - would you like to test that out?